### PR TITLE
fixed team creation response if teamId already exists

### PR DESCRIPTION
**Pull request by jaivix**
## What does this PR do?

Fixes the linked issue 

## Test Plan
- Created a simple web project, tried to create team with same teamId - 
image

- Wrote test case for this - 
image

## Related PRs and Issues

- https://github.com/appwrite/appwrite/issues/5768

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: src/Appwrite/Extend/Exception.php [hunk 1]
```diff
@@ -83,6 +83,7 @@ class Exception extends \Exception
     public const TEAM_INVALID_SECRET               = 'team_invalid_secret';
     public const TEAM_MEMBERSHIP_MISMATCH          = 'team_membership_mismatch';
     public const TEAM_INVITE_MISMATCH              = 'team_invite_mismatch';
+    public const TEAM_ID_CONFLICT                  = 'team_id_conflict';
```
Comment (stnguyen90) [inline src/Appwrite/Extend/Exception.php:86]: It's always good to take a look at existing code to try and match existing conventions. In this case, for other similar errors, we have `<RESOURCE>_ALREADY_EXISTS` such as `TEAM_INVITE_ALREADY_EXISTS`. So, this should probably be:

```suggestion
    public const TEAM_ALREADY_EXISTS                = 'team_already_exists';
```

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

Also, it looks like the linter failed. Please run the formatter to fix the lint error.

**Comment by jaivix** [comment]
not sure why test cases were failing since locally test cases seems to run fine, maybe because the repo wasn't synced with appwrite master, synced with master so this time the test cases should pass hopefully 🤞 
@stnguyen90 can you please review again.

**Review by eldadfux — APPROVED** [review]

**Comment by jaivix** [comment]
thanks 🙏

**Pull request merged**